prez/routers/management.py

```diff
@@ -5,19 +5,21 @@
 
 from prez.cache import tbox_cache
 from prez.renderers.renderer import return_rdf
+from prez.services.app_service import add_common_context_ontologies_to_tbox_cache
 
 router = APIRouter(tags=["Management"])
 
 
-@router.get("/purge-tbox-cache", summary="Purge Tbox Cache")
-async def purge_tbox_cache():
-    """Purges the Tbox cache"""
+@router.get("/reset-tbox-cache", summary="Reset Tbox Cache")
```

Comment:
Or if we intend to release it as breaking, outline it in the README and ensure that it is stated as breaking in the release notes.

Reply:
I'll just mark it breaking unless there's a desire to have both purge (remove all) and reset (remove all, then add back in annotations from common ontologies included w/ prez)

Reply:
If they're the same and going forward, we always add in the common ontologies bundled in Prez, then do we really need an API name change here?

We need to be careful with breaking changes as there are more and more deployments of Prez where automated pipelines rely on hitting these APIs for their workflows.

Reply:
the intention is you'd use reset instead of purge yes, the point of changing the name was to reflect the behaviour change. Happy to change back to "purge" or could provide both routes and eventually deprecate the purge one? Would you prefer one of these options?

Reply:
IMO, the name with "purge" is fine. Not sure what others think.

I don't really see it as a behaviour change since we're still _purging the data cached from the triplestore_. The base state of the cache now just comes with the preloaded common ontologies.

Reply:
Have changed back